ResourceContainer.gd

```diff
@@ -20,16 +22,16 @@ signal updated_container(
     inventory_name: String, 
 )
 signal transaction_executed(
-    transaction: TransactionResource, 
-    exit_status: TransactionResource.ExitStatus
+    transaction: Transaction, 
+    exit_status: Transaction.ExitStatus
 )
 
 func get_container():
     return container
 
 func create_container():
 
-    container = ContainerResource.new(
+    container = Store.new(
```

Comment:
Calling this a "store" instead would be more consistent.

A ContainerResource uses a "store" so it has a "store" property.

Reply:
that makes sense to me. I waffled on that one for a minute and so just left it. But hearing you say this also gives me confidence to make the change.